test/widgets/compose_box_test.dart

```diff
@@ -726,10 +777,21 @@ void main() {
         expectedMessage: 'Topics are required in this organization.');
     }
 
-    testWidgets('empty topic -> "(no topic)"', (tester) async {
+    testWidgets('empty topic -> empty topic', (tester) async {
```

Comment:
How about calling this 'empty topic -> ""'? The left side is about the state of a text input, and the right is about a literal string, right?